ccmbioinfo/lfs
# CARDS-2095: Clarity Import should use the HOSP_DISCHARGE_DTTM column for the date to load filter

To test:
- build with `-Pdocker`
- in `compose-cluster` run `python3 generate_compose_yaml.py --mssql --cards_project cards4prems --oak_filesystem --dev_docker_image --smtps --smtps_test_container --smtps_test_mail_path ~/cardsmail --composum --adminer`
- start with `docker-compose build && docker-compose up`
- at `http://localhost:1435` log in and import `compose-cluster/mssql/processorImportTestSql.sql`
- trigger an import at `localhost:8080/Subjects.importClarity?pastDayToQuery=100`, wait a bit and make sure nothing was imported
- trigger an import at `localhost:8080/Subjects.importClarity?pastDayToQuery=0` (adjust with the number of days since 2023-02-20), wait a bit, make sure the visits were imported
